todolist copy 3/main.js

```diff
@@ -1,111 +1,134 @@
-	checkBoxComplete.onclick = function() {completeTask(index)};
+      const taskLabel = document.createElement('label')
+      taskLabel.classList.add('tasks')
 
-	let taskName = document.createElement('span');
-	taskName.classList.add("task-name");
-	taskName.innerHTML = item.description;
+      const checkBoxComplete = tickCheckbox(item, index)
 
-	let checkMark = document.createElement('span');
-	checkMark.classList.add("checkmark");
+      const taskName = createTask(item)
 
+      const checkMark = document.createElement('span')
+      checkMark.classList.add('checkmark')
 
-	taskLabel.appendChild(checkBoxComplete);
-	taskLabel.appendChild(taskName);
-	taskLabel.appendChild(checkMark);
+      taskLabel.appendChild(checkBoxComplete)
+      taskLabel.appendChild(taskName)
+      taskLabel.appendChild(checkMark)
 
-	taskLine.appendChild(deleteButton);
-	taskLine.appendChild(taskLabel);
+      taskLine.appendChild(deleteButton)
+      taskLine.appendChild(taskLabel)
 
-	toDoList.appendChild(taskLine);
-	
-			})
-			toDoElem = document.querySelectorAll('.task-item'); 
-		}
+      toDoList.appendChild(taskLine)
+    })
+    toDoElem = document.querySelectorAll('.task-item')
+  };
 }
-updateHTML();
-	const updateLocal = () => {
-	localStorage.setItem('tasks', JSON.stringify(tasks));
+
+const updateLocal = () => {
+  localStorage.setItem('tasks', JSON.stringify(tasks))
+  tasks.sort(function (a, b) {
+    return a.completed - b.completed
+  })
 }
 
 const completeTask = index => {
-	tasks[index].completed = !tasks[index].completed; 
-	if(tasks[index].completed) {
-		toDoElem[index].classList.add('done'); 
-	}
-	else {
-		toDoElem[index].classList.remove('done');
-	}
-	updateLocal();
-	updateHTML();
+  tasks[index].completed = !tasks[index].completed
+  updateLocal()
+  updateList()
 }
 
 addTaskButton.addEventListener('click', () => {
-	tasks.push(new Task(lineTaskInput.value));
-	updateLocal();
-	updateHTML();
-	lineTaskInput.value = '';
-});
+  tasks.push(new Task(lineTaskInput.value))
+  updateLocal()
+  updateList()
+  lineTaskInput.value = ''
+})
 
 const deleteTask = index => {
-	tasks.splice(index, 1);
-	updateLocal();
-	updateHTML();
+  tasks.splice(index, 1)
+  updateLocal()
+  updateList()
 }
 
-clearing.addEventListener("click", () => {
-	tasks = [];
-	updateLocal();
-	updateHTML();
-});
+clearing.addEventListener('click', () => {
+  tasks = []
+  updateLocal()
+  updateList()
+})
 
 document.querySelector('#taskInput').addEventListener('keypress', function (e) {
-    if (e.key === 'Enter') {
-      tasks.push(new Task(lineTaskInput.value));
-	updateLocal();
-	updateHTML();
-	lineTaskInput.value = '';
-    }
-});
+  if (e.key === 'Enter') {
+    tasks.push(new Task(lineTaskInput.value))
+    updateLocal()
+    updateList()
+    lineTaskInput.value = ''
+  }
+})
+
+function completedItem (item) {
+  const taskLine = document.createElement('li')
+  taskLine.classList.add('task-item')
+  if (item.completed) {

```

Comment:
We can do it as 1-liner `taskLine.classList.toggle('done', item.completed)`. No `if` required.